### PR TITLE
fix: revert back to commons js

### LINKED ISSUE #1969: Fix CJS imports
Hey guys so unfortunately I think this still doesn't work. Or it's also entirely possible I'm doing something wrong but, whenever I try to import something from `ts-json-schema-generator` I get this error: 
```shell
Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/stevencummings/workspace/ternary/core-web-server/node_modules/ts-json-schema-generator/cjs/index.js from /Users/stevencummings/workspace/ternary/core-web-server/scripts/docs-gen/Oas3Generator.ts not supported.
index.js is treated as an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which declares all .js files in that package scope as ES modules.
Instead either rename index.js to end in .cjs, change the requiring code to use dynamic import() which is available in all CommonJS modules, or change "type": "module" to "type": "commonjs" in /Users/stevencummings/workspace/ternary/core-web-server/node_modules/ts-json-schema-generator/package.json to treat all .js files as CommonJS (using .mjs for all ES modules instead).
```

Sorry if this isn't the best place to discuss, lmk if I should open a new issue or what the best thing is to do.

_Originally posted by @sacummings91 in https://github.com/vega/ts-json-schema-generator/issues/1964#issuecomment-2130251619_

### PR DESCRIPTION
**Pull request by arthurfiorette**
<details>
  <summary>📦 Published PR as canary version: <code>2.3.0--canary.1975.2880903.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install ts-json-schema-generator@2.3.0--canary.1975.2880903.0
  # or 
  yarn add ts-json-schema-generator@2.3.0--canary.1975.2880903.0
  ```
</details>

# Version

Published prerelease version: `v2.3.0-next.2`

<details>
  <summary>Changelog</summary>

  #### 🚀 Enhancement
  
  - fix: revert back to commons js [#1975](https://github.com/vega/ts-json-schema-generator/pull/1975) ([@arthurfiorette](https://github.com/arthurfiorette))
  - fix: remove ts-expose-types temporarily [#1968](https://github.com/vega/ts-json-schema-generator/pull/1968) ([@arthurfiorette](https://github.com/arthurfiorette))
  - Added diagnostics to all errors [#1963](https://github.com/vega/ts-json-schema-generator/pull/1963) ([@arthurfiorette](https://github.com/arthurfiorette))
  
  #### 🔩 Dependency Updates
  
  - chore(deps-dev): bump eslint from 9.2.0 to 9.3.0 [#1970](https://github.com/vega/ts-json-schema-generator/pull/1970) ([@dependabot[bot]](https://github.com/dependabot[bot]))
  - chore(deps-dev): bump ajv from 8.13.0 to 8.14.0 [#1971](https://github.com/vega/ts-json-schema-generator/pull/1971) ([@dependabot[bot]](https://github.com/dependabot[bot]))
  - chore(deps-dev): bump typescript-eslint from 7.8.0 to 7.10.0 [#1972](https://github.com/vega/ts-json-schema-generator/pull/1972) ([@dependabot[bot]](https://github.com/dependabot[bot]))
  
  #### Authors: 2
  
  - [@dependabot[bot]](https://github.com/dependabot[bot])
  - Arthur Fiorette ([@arthurfiorette](https://github.com/arthurfiorette))
</details>

### CODE REVIEW

**Comment by arthurfiorette** [comment]
Hey @sacummings91 and @navalex could you guys try out `2.2.1--canary.1975.4fce06c.0`?

**Comment by arthurfiorette** [comment]
@domoritz ready for review

File: eslint.config.js
Comment (domoritz) [inline eslint.config.js:1]: can we use `eslint.config.mjs`?

**Review by domoritz — APPROVED** [review]

File: eslint.config.js
Comment (arthurfiorette) [reply eslint.config.js:1]: I couldn't get it to work :/

Comment (domoritz) [reply eslint.config.js:1]: What's the error?

**Comment by navalex** [comment]
> Hey @sacummings91 and @navalex could you guys try out `2.2.1--canary.1975.4fce06c.0`?

Done, reply in my issue

**Comment by arthurfiorette** [comment]
The Commonsjs bundle worked. Your issue has another error now.

**Pull request merged**